rust/syntect-plugin/src/main.rs

```diff
@@ -277,25 +277,28 @@ impl<'a> Syntect<'a> {
         let tab_size = view.get_config().tab_size;
         let current_indent = self.indent_level_of_line(view, line);
         let base_indent = self
-            .previous_nonblank_line(view, line)?
+            .previous_line(view, line)?
             .map(|l| self.indent_level_of_line(view, l))
             .unwrap_or(0);
+
         let increase_level = self.test_increase(view, line)?;
         let indent_level = if increase_level { base_indent + tab_size } else { base_indent };
+
```

Comment:
If I test here whether it should be decreased with `test_decrease` it returns an error (`error in autoindent PeerDisconnect`). The reason for this seems to be that at some point in `get_metadata` were are trying to get the `line` with `view.get(line)`. View seems to access some cache with the lines but cannot find `line` and returns an error. I am not sure if that is the expected behaviour?